pkg/fleet/installer/packages/embedded/templates/gen/debrpm/datadog-agent-ddot-exp.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Datadog Distribution of OpenTelemetry Collector Experiment
+After=datadog-agent.service datadog-agent-exp.service
```

Comment:
Does it make sense to have `datadog-agent.service` here if it conflicts with it? Also, if you read my comment below, it's important to determine if `datadog-agent.service` is a real conflict.